components/ai_chat/common/mojom/ai_chat.mojom

```diff
@@ -80,6 +82,10 @@ struct Model {
   // user-facing category
   ModelCategory category;
   bool is_premium;
+  // max limit to truncate page contents
+  uint32 max_page_content_length;
+  // max limit for overall conversation
```

Comment:
stress that this is approximate and in characters (not tokens)

Reply:
added in the comment